src/Peering/Peering/help/Remove-AzPeeringServicePrefix.md

```diff
@@ -0,0 +1,247 @@
+---
```

Comment:
@rileymckenna once all the above request changes are made, you should do another generation of the markdown files with the below command:

```powershell
Update-MarkdownHelpModule -Path {{PATH_TO_HELP}} -RefreshModulePage -AlphabeticParamsOrder -UseFullTypeName -UpdateInputOutput
```